### PR TITLE
Fix startup.cmd and tabpy.py

### PR DESCRIPTION
**Pull request by 0golovatyi**

### CODE REVIEW

File: startup.cmd [hunk 1]
```diff
@@ -43,33 +46,31 @@ IF %ERRORLEVEL% NEQ 0 (
 
 REM Parse optional CLI arguments: config file
 ECHO Parsing parameters...
-SET STARTUP_CMD=python tabpy-server\tabpy.py
+SET PYTHONPATH=%TABPY_ROOT%\tabpy-server;%PATH%
```
Comment (johng42) [inline startup.cmd]: Is this even used?

File: tabpy-server/tabpy_server/tabpy.py [hunk 2]
```diff
@@ -1,967 +1,10 @@
...
-import multiprocessing
-from OpenSSL import crypto
-import os
-from pathlib import Path
-from re import compile as _compile
-import requests
-import shutil
-import simplejson
-import sys
 from tabpy_server import __version__
```
Comment (johng42) [inline tabpy-server/tabpy_server/tabpy.py]: Is this file change related to the rest of the fix?  It's really hard to tell from this one change.

File: startup.cmd [hunk 3]
```diff
@@ -2,18 +2,20 @@
 SETLOCAL
 
 
-REM Check for Python in PATH
+REM Set environment variables.
+SET TABPY_ROOT=%CD%
+SET INSTALL_LOG=%TABPY_ROOT%\tabpy-server\install.log
```
Comment (johng42) [inline startup.cmd]: Is this ever used?

File: startup.cmd (hunk 1 above)
Comment (0golovatyi) [reply startup.cmd]: Used by Python when searching packages

File: startup.cmd (hunk 3 above)
Comment (0golovatyi) [reply startup.cmd]: Used to append to log file
